packages/ignition/src/app/layout.tsx

```diff
@@ -29,7 +29,7 @@ const inter = Inter({
 });
 
 export const generateMetadata = ({ params: { lang } }: PropsWithLangParams): Metadata => {
-  return customMetadata(lang, "page");
+  return customMetadata(lang, "page", "");
```

Comment:
I don't think we need to pass an empty arg here. If it's possible, I think a better solution is making this parameter optional